src/couch/src/couch_util.erl

```diff
@@ -829,3 +831,32 @@ hex(X) ->
         16#6530, 16#6531, 16#6532, 16#6533, 16#6534, 16#6535, 16#6536, 16#6537, 16#6538, 16#6539, 16#6561, 16#6562, 16#6563, 16#6564, 16#6565, 16#6566,
         16#6630, 16#6631, 16#6632, 16#6633, 16#6634, 16#6635, 16#6636, 16#6637, 16#6638, 16#6639, 16#6661, 16#6662, 16#6663, 16#6664, 16#6665, 16#6666
     }).
+
+verify_hash_names(HashAlgorithms, SupportedHashFun) ->
```

Comment:
A tiny nit:  The `SupportedHashFun` name, at first sight, implies this is a callable function, but it is actually a list of hashes. Should we rename to something like `SupportedHashes`?

Reply:
Oh yes, will change this!